### ISSUE TITLE
Disable triangle lines on the surface

### ISSUE THREAD

**Issue by foldfelis** [JuliaPlots/Plots.jl#3279]
https://github.com/JuliaPlots/Plots.jl/blob/781e6d2a1623249eca413538380920d5ef1c127d/src/backends/gr.jl#L1739

According to the source code, when I set `st=:surface`, it will call the API `GR.trisurface`. Is there any way to plot a 3D surface without the triangle lines?

![image](https://user-images.githubusercontent.com/59465497/107157579-c0d84100-69bf-11eb-8350-b7f31764b946.png)

**Comment by foldfelis** [comment]
Hello, @daschw , 

The situation I ran into is that I already has a group of points and would like to draw them as a surface.

I have tried drawing with `GR`  via [this API](https://gr-framework.org/python-gr.html?highlight=surface#gr.surface), and it works just fine. But in Plots, when `length(x) == length(y) == length(z)`, it will use `GR.trisurface(x, y, z)` instead of `GR.surface(x, y, z)`.

is there any way to plot a group of points as a surface by Plots?
